python/src/MarshallOlkinCopula_doc.i.in

```diff
@@ -0,0 +1,78 @@
+%feature("docstring") OT::MarshallOlkinCopula
+"MarshallOlkin copula.
+
+Available constructor:
+    MarshallOlkinCopula(*alpha=0.0,beta=0.0*)
```

Comment:
No need to list the available ctors when there's only one.

Reply:
Ok.